### ISSUE TITLE
rmarkdown::render complains about missing hs_chapters

### ISSUE THREAD

**Issue by chrmongeau** [SWS-Methodology/faoswsTrade#99]
`rmarkdown::render('main.R')` (on *complete_tf_cpc/* directory) gives the following error:

```
Quitting from lines 212-213 (main.spin.Rmd) 
Error in formatC(hs_chapters, width = 2, format = "d", flag = "0") : 
  object 'hs_chapters' not found
```

**Comment by chrmongeau** [comment]
While this is a bug, it is not critical as long as a new version of the module's documentation needs to be produced.

**Comment by chrmongeau** [comment]
Fixed by 29e3339.
